src/router/index.js

```diff
@@ -67,12 +67,6 @@ export default new Router({
           name: 'Login',
           component: Login,
         },
-        {
```

Comment:
Was this only used for testing purpose ?

Reply:
Yes, removal requested by @DanielBrierton since it's not needed anymore and causes issues with url queries